support-frontend/assets/pages/[countryGroupId]/thankYou.tsx

```diff
@@ -60,6 +61,8 @@ export function ThankYou({ geoId, appConfig }: ThankYouProps) {
 			originalAmount: finalAmount,
 			finalAmount: finalAmount,
 		};
+
+		setHideSupportMessagingCookie();
```

Comment:
Is this part of this PR specifically? If not, maybe a comment might be good here?